### PR TITLE
Remove extra send/receive annotations with using zipkin v1

### PR DESCRIPTION
**Pull request by chris-smith-zocdoc**
Fixes https://github.com/open-telemetry/opentelemetry-collector/issues/960

### CODE REVIEW

File: translator/trace/zipkin/zipkinv1_thrift_to_protospan_test.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package zipkin
 import (
 	"encoding/binary"
 	"encoding/json"
+	"github.com/google/go-cmp/cmp"
```
Comment (chris-smith-zocdoc) [inline translator/trace/zipkin/zipkinv1_thrift_to_protospan_test.go:20]: I pulled in go-cmp because it outputs really nice diffs unlike reflect.DeepEqual

Comment (bogdandrutu) [reply translator/trace/zipkin/zipkinv1_thrift_to_protospan_test.go:20]: Use `assert.EqualValues` as we do in the rest of the repo.

Comment (chris-smith-zocdoc) [reply translator/trace/zipkin/zipkinv1_thrift_to_protospan_test.go]: Updated for consistency. go-cmp does produce better outputs imo though

**Comment by dmitryax** [comment]
@chris-smith-zocdoc please sign the Contributor License Agreement

**Comment by chris-smith-zocdoc** [comment]
I signed it

**Comment by chris-smith-zocdoc** [comment]
Was there anything else blocking this?

**Pull request merged**